README.md

```diff
@@ -20,26 +20,81 @@ pip install personal-graph
 
 ## Usage
 
+### Usage of Vector store classes
+There are two vector store classes SQLiteVSS and VLiteDatabase. Let's dive down into the implementation of both the vector store classes.
+
+
+```
+from personal_graph.database import SQLiteVSS, DBClient, TursoDB
+from personal_graph import EmbeddingClient
+
+# Using SQLiteVSS vector database and connecting with Turso DB
+vector_store = SQLiteVSS(
+        persistence_layer=TursoDB(
+            db_client=DBClient(
+                db_url=args.db_url,
+                db_auth_token=args.db_auth_token,
+            ),
+            embedding_model_client=EmbeddingClient(),
+        )
+    )
+
+# Using SQLiteVSS vector database and connecting with local DB
+vector_store = SQLiteVSS(
+        persistence_layer=SQLite(
+            db_client=DBClient(
+                use_in_memory=True,
+                vector0_so_path="path/to/vector0.so",
+                vss0_so_path="path/to/vss0.so"
+            ),
+            embedding_model_client=EmbeddingClient(),
+        )
+    )
+
+# Using SQLiteVSS vector database and connecting with in-memory DB
+vector_store = SQLiteVSS(
+        persistence_layer=SQLite(
+            db_client=DBClient(
+                use_in_memory=True,
+                vector0_so_path="path/to/vector0.so",
+                vss0_so_path="path/to/vss0.so"
+            ),
+            embedding_model_client=EmbeddingClient(),
+        )
+    )
+
+# Using VLite vector database 
+from personal_graph.database import VLiteDatabase
+
+vector_store = VLiteDatabase(collection="your-collection-name", model_name="embedding-model-name")
+
+```
+
 ### Building a Working Memory for an AI
 
 ```python
-from personal_graph import Graph
+from personal_graph import Graph, LLMClient
+from personal_graph.database import SQLiteVSS, DBClient, TursoDB
+from personal_graph.graph_generator import InstructorGraphGenerator
+
 
-graph = Graph("your_db_url", "your_auth_token")
+graph = Graph(vector_store=vector_store, graph_generator=InstructorGraphGenerator(llm_client=LLMClient()))
```

Comment:
the InstructorGraphGenerator should be the default value for graph_generator, so then it wont be necessary to pass it in here